Riot/Modules/SetPinCode/PinCodePreferences.swift

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2018 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+/// Pin code preferences.
+@objcMembers
+final class PinCodePreferences: NSObject {
+    
+    // MARK: - Constants
+    
+    private enum StoreKeys {
+        static let pin = "pin"
+    }
+    
+    static let shared = PinCodePreferences()
+    
+    /// Store. Defaults to `KeychainStore`
+    var store: KeyValueStore!
+    
+    override init() {
+        store = KeychainStore()
```

Comment:
In this case, call `super.init()` at the end of the init.

Reply:
👍